packages/odyssey/src/scss/abstracts/_mixins.scss

```diff
@@ -23,9 +23,9 @@
   border: 0;
 }
 
-@mixin outline($outlineColor: 'action', $radius: 4px) {
+@mixin outline($outlineCv: $focus-ring-primary, $radius: 4px) {
```

Comment:
can we keep our function/mixin variable naming matched to our "normal" variable naming? `$x-y`